test/e2e/apps/statefulset.go

```diff
@@ -461,6 +461,58 @@ var _ = SIGDescribe("StatefulSet", func() {
 				}
 			}
 
+			ginkgo.By("Restoring Pod-0 to the correct revision when all of the old pod are deleted")
+			ss.Spec.UpdateStrategy = appsv1.StatefulSetUpdateStrategy{
+				Type: appsv1.RollingUpdateStatefulSetStrategyType,
+				RollingUpdate: func() *appsv1.RollingUpdateStatefulSetStrategy {
+					return &appsv1.RollingUpdateStatefulSetStrategy{
+						Partition: pointer.Int32(1),
+					}
+				}(),
+			}
+			ss, err = updateStatefulSetWithRetries(ctx, c, ns, ss.Name, func(update *appsv1.StatefulSet) {
+				update.Spec.UpdateStrategy = appsv1.StatefulSetUpdateStrategy{
+					Type: appsv1.RollingUpdateStatefulSetStrategyType,
+					RollingUpdate: func() *appsv1.RollingUpdateStatefulSetStrategy {
+						return &appsv1.RollingUpdateStatefulSetStrategy{
+							Partition: pointer.Int32(1),
+						}
+					}(),
+				}
+			})
+			framework.ExpectNoError(err)
```

Comment:
```suggestion
			gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Reply:
Thanks for review, this change have update on the other PR, so i have not update it.

Reply:
tests are not going to pass if you don't change it. lint tests are going to fail again.